### PR TITLE
feat: setup min value that can be transferred

### PR DESCRIPTION
**Pull request by manh-vv**

### CODE REVIEW

**Review by manh-vv — COMMENTED** [review]
Ready for reviewing.

File: src/deposit_screen.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+   _global.set(gstate, get_self());
+}
+
+void deposit_screen::on_wax_transfer(
+    const name &from,
+    const name &to,
+    const asset &quantity,
+    const string &memo)
+{
+   auto gstate = _global.get();
```
Comment (porkchop) [inline src/deposit_screen.cpp:34]: @manh-vv I think we might want to make some minimum verifications here as well:
```
    check(get_first_receiver() == "eosio.token"_n, "Invalid token contract");

    if (from == get_self() || to != get_self()) {
      return;
    }

    check(quantity.symbol == WAX_SYMBOL, "Invalid token");
```

**Review by porkchop — CHANGES_REQUESTED** [review]
Looking good @manh-vv I see only one issue to fix up

File: src/deposit_screen.cpp (hunk 1 above)
Comment (manh-vv) [reply src/deposit_screen.cpp:34]: Thank @porkchop .
I applied your comment.

**Review by manh-vv — COMMENTED** [review]
applied comment

**Review by porkchop — APPROVED** [review]
LGTM Thanks @manh-vv

**Pull request merged**